### ISSUE TITLE
Update dates facets to include start and end dates (from 1961)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#69]
**Problem Description:**
The facets for dates only return start dates.

**Expected Behavior/Solution:**
Update the facets to include end dates based on the ML data returned.

**Requirements:**
- Facets will include end dates from the ML data returned. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] ~Wireframe/Mockup - Heather~
- [ ] ~Committee discussions - Sarah~
- [ ] ~Feasibility/Team discussion - Sarah~
- [x] Backend requirements - See Dependencies below - Peter
- [ ] ~Frontend requirements~
- [x] Questions

**UAT/LUX Examples:**

- https://lux.collections.yale.edu/view/results/objects/?q=%7B%22text%22%3A%22equiano%22%2C%22_lang%22%3A%22en%22%7D&sq=equiano&is=itemProductionDate%3Adesc

**Dependencies/Blocks:**

- [x] Blocked by https://github.com/project-lux/lux-marklogic/issues/43 & 
- [x] https://github.com/project-lux/lux-marklogic/issues/37

**Related Github Issues:**
N/A

**Related links:**
Bugherd: https://www.bugherd.com/projects/284041/tasks/1245

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by roamye** [comment]
From ITS TEAM Meetin 8/23: 
- This is blocked by ML 43.

**Comment by roamye** [comment]
This is no longer blocked by ML43 as it was closed. Moved this to backlog. 

I wonder if this is still needed since we have the new dates functionality? 
@kamerynB

**Comment by roamye** [comment]
from ticket forming mtg 10/08: 

this issue is still blocked & blocked by: https://github.com/project-lux/lux-marklogic/issues/37

**Comment by clarkepeterf** [comment]
This can be closed - no longer needed after Optic updates
